src/scib_metrics/utils/_utils.py

```diff
@@ -63,3 +63,52 @@ def convert_knn_graph_to_idx(X: csr_matrix) -> tuple[np.ndarray, np.ndarray]:
         nn_obj = NearestNeighbors(n_neighbors=n_neighbors, metric="precomputed").fit(X)
         kneighbors = nn_obj.kneighbors(X)
     return kneighbors
+
+
+def compute_connectivities_umap(
+    knn_indices,
+    knn_dists,
+    n_obs,
+    n_neighbors,
+    set_op_mix_ratio=1.0,
+    local_connectivity=1.0,
+):
+    """Sped up version of sc.neighbors._compute_connectivities_umap."""
```

Comment:
Can you put a more general docstring? Overview of the method and that it matches how connectivies are computed in scanpy?